<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Images/extras_configure.png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidgetSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="General">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>280</width>
         <height>114</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutRasp">
        <item>
         <widget class="QLabel" name="lblRaspberryPI">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Raspberry PI</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRasp">
          <item>
           <layout class="QVBoxLayout" name="fieldRaspAddress">
            <item>
             <widget class="QLabel" name="lblRaspAddress">
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtRaspAddress">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="fieldRaspPort">
            <item>
             <widget class="QLabel" name="lblRaspPort">
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtRaspPort">
              <property name="text">
               <string>1234</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnectRasp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/Images/RASPBERRY_PI_2_B_01.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Servers">
      <attribute name="title">
       <string>Servers</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>280</width>
         <height>115</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="fieldAddress">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="labelAddress">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtHost">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="fieldPort">
            <item>
             <widget class="QLabel" name="labelPort">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtPort">
              <property name="text">
               <string>5250</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="fieldOscPort">
            <item>
             <widget class="QLabel" name="lblOscPort">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>OSC Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtOScPort">
              <property name="text">
               <string>6250</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chkAutoConnect">
          <property name="text">
           <string>Connect automatically</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTreeWidget" name="treeWidgetDevice">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>551</width>
         <height>321</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Icon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Username</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Password</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Version</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Shadow</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Channels</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Channel Formats</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Preview Channel</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Locked Channel</string>
        </property>
       </column>
      </widget>
      <widget class="QToolButton" name="toolButtonAddDevice">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>510</y>
         <width>33</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Add CasparCG Server</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Graphics/Images/Add.png</normalon>
        </iconset>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButtonRemoveDevice">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>510</y>
         <width>33</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove CasparCG Server</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Graphics/Images/Remove.png</normalon>
        </iconset>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Midi">
      <attribute name="title">
       <string>MIDI</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>411</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inPortLayout">
        <item>
         <widget class="QLabel" name="inPortLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MIDI In Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="inPortComboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>411</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="outPortLayout">
        <item>
         <widget class="QLabel" name="outPortLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MIDI Out Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="outPortComboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonAddDevice</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>showAddDeviceDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRemoveDevice</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>removeDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetDevice</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>deviceItemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAddDeviceDialog()</slot>
  <slot>removeDevice()</slot>
  <slot>deviceItemDoubleClicked(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
